ide/gen-stubs.php

```
@@ -3,21 +3,30 @@
 /**
  * This scripts generates the stubs to be used on IDEs
  *
- * Change the CPHALCON_DIR constant to point to the dev/ directory in the Phalcon source code
+ * Specify CPHALCON_DIR env variable to point to the dev/ directory in the Phalcon source code
  *
- * php ide/gen-stubs.php
+ * *nix:    CPHALCON_DIR=/path/to/cphalcon/ext/ php ide/gen-stubs.php
+ *  Win:    set CPHALCON_DIR=C:\Path\To\cphalcon\ext\ php ide/gen-stubs.php
+ *
+ * Alternative stubs generation
+ *
+ * cd cphalcon
+ * zephir stubs
  */
 
 if (!extension_loaded('phalcon')) {
 	throw new Exception("phalcon extension isn't installed");
 }
 
-define('CPHALCON_DIR' , '/Users/micate/Code/cphalcon/ext/');
-
-if (!file_exists(CPHALCON_DIR)) {
+if (getenv('CPHALCON_DIR') === false || !file_exists(getenv('CPHALCON_DIR'))) {
```

@Jeckerson can you please throw a separate exception when the environment variable doesn't exist and have the message explain where it should point?